INSS.EIIR.Web/Views/SearchDetails/_DRRUDetail.cshtml

```diff
@@ -0,0 +1,61 @@
+﻿@using INSS.EIIR.Web.Constants
+@using INSS.EIIR.Web.Extensions
+@model INSS.EIIR.Models.CaseModels.CaseResult
+
+@*
```

Comment:
remove this line  : For more information on enabling MVC for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860